sys/lpeops/src/core/LPEContextIds.hs

```diff
@@ -127,7 +127,6 @@ getValExprIds = customData . visitValExpr searchVisitor
                     (view -> Vconcat _)               -> idsInSubExps
                     (view -> Vstrinre _ _)            -> idsInSubExps
                     (view -> Vpredef _ fid _)         -> Set.insert (TxsDefs.IdFunc fid) idsInSubExps
-                    _                                 -> error ("GetValExprIds.searchVisitor not defined for " ++ show expr ++ "!")
```

Comment:
Why did you remove these error messages to the user?
LPE doesn't support all construct (e.g. not dynamic process creation),
user should be informed when he is using LPE incorrectly.